tokens/blau.json

```diff
@@ -25,6 +25,11 @@
       "type": "color",
       "description": "white"
     },
+    "backgroundContainerError": {
+      "value": "{palette.white}",
```

Comment:
sorry, I can't understand why "white" is set up for this token for all brands. Shouldn't it be equivalent to `errorLow`?

![image](https://github.com/Telefonica/mistica-design/assets/6722153/6aae4276-c580-4837-a965-76496bef3a4f)

This comment apply to all skins

Reply:
should be the same color in backgroundContainerError and errorLow, you're right! I'll change it

Reply:
You didn't change it! xD

I think you make changes in dark mode and not in light mode